examples/shaders/resources/shaders/glsl330/lighting_instancing.vs

```diff
@@ -25,8 +25,8 @@ void main()
     // Send vertex attributes to fragment shader
     fragPosition = vec3(instanceTransform*vec4(vertexPosition, 1.0));
     fragTexCoord = vertexTexCoord;
-    //fragColor = vertexColor;
-    fragNormal = normalize(vec3(matNormal*vec4(vertexNormal, 1.0)));
+    fragColor = vec4(1.0);
+    fragNormal = normalize(vec3(matNormal * vec4(vertexNormal, 1.0)));
```

Comment:
Please maintain the [raylib coding conventions](https://github.com/raysan5/raylib/wiki/raylib-coding-conventions).